tests/acceptance/manager/PartsCest.php

```diff
@@ -155,4 +208,18 @@ protected function getPartData(): array
             'company_id'    => 'Other'
         ];
     }
+
+    protected function getSellData(): array
+    {
+        return [
+            'sellInfo' => [
+                'contact_id'=> 'Test Manager',
+                'currency'  => 'eur',
+                'descr'     => 'test description ' . uniqid(),
+                'type'      => 'HW purchase',
+                'prices'    => [250, 300, 442],
+                'time'      => (new \DateTime())->format('Y-m-d H:i'),
```

Comment:
```suggestion
                'time'      => (new DateTime())->format('Y-m-d H:i'),
```
а зверху `use DateTime;`

Reply:
Не працюэ без бэкслеша

Reply:
> а зверху use DateTime;

вот же в коменті вище писав